### PR TITLE
Workflow Tests: Add Slugs to Translation Queue

### PR DESCRIPTION
**Pull request by zstix**
## Description
In an effort to make our translation workflows feel more stable, I wanted to take some time to write proper tests for each workflow. This is the first in a series of PRs (one workflow at a time). My goal is to establish some helpful code references for things like mocking calls to our DB, faking a file system for use in tests, etc.

## Reviewer Notes
* I'm still adding some test cases (for edge-case scenarios)
* I think I am starting to uncover bugs with the code, but I wanted to review it with the team first

### CODE REVIEW

File: scripts/actions/__tests__/add-files-to-translation-queue.test.js [hunk 1]
```diff
@@ -0,0 +1,256 @@
+import fs from 'fs';
+import AWS from 'aws-sdk';
+import fetch from 'node-fetch';
```
Comment (aswanson-nr) [inline scripts/actions/__tests__/add-files-to-translation-queue.test.js:3]: I think we'll need to change these imports over to the old `require` syntax, since this will be running on node 12 in the Github runners.

File: scripts/actions/add-files-to-translation-queue.js [hunk 2]
```diff
@@ -84,7 +85,20 @@ const main = async () => {
 
   await saveToTranslationQueue(key, 'set locales = :slugs', { ':slugs': data });
 
-  process.exit(0);
+  // If we are in a CI environment, return a success code
+  if (process.env.CI) {
```
Comment (aswanson-nr) [inline scripts/actions/add-files-to-translation-queue.js:89]: Github Actions automatically set this to `true` and it doesn't look like you can override it. 
You can use `require.main === module` instead, it will evaluate to `true` if the file is executed by Node and `false` if the file is imported by another module. https://nodejs.org/docs/latest/api/modules.html#modules_accessing_the_main_module

**Comment by jpvajda** [comment]
Moving back to in-progress as there are some suggestions, probably best to wait for @zstix and @aswanson-nr to return

**Comment by jpvajda** [comment]
Just to confirm: is this the remainder of the work we need to complete before we can turn back on the [translation workflow job?](https://github.com/newrelic/docs-website/actions/workflows/send-content-to-translate.yml)

**Comment by zstix** [comment]
I'm closing this because it's a little too stale. Going to leave the branch so that we can use it for reference.
